### PR TITLE
Display message template title on deletion form

### PR DESCRIPTION
**Pull request by JKingsnorth**
Overview
----------------------------------------
On the deletion confirmation page for a message template it doesn't display the name of the template being deleted. It would be better UX to confirm the name of the template being deleted, otherwise it rather defeats the object of the confirmation page! ;]

Steps to recreate:
- Mailingg > Message templates > Choose one to delete

Before
----------------------------------------
image

After
----------------------------------------
image

Technical Details
----------------------------------------
-  Assign the message title to a template variable for the delete action
- Add the message title into the text on the page, in a translatable string

### CODE REVIEW

File: templates/CRM/Admin/Form/MessageTemplates.tpl [hunk 1]
```diff
@@ -39,7 +39,7 @@
 {if $action eq 8}
    <div class="messages status no-popup">
        <div class="icon inform-icon"></div>
-       {ts}Do you want to delete this message template?{/ts}
+       {ts 1=$msg_title}Do you want to delete the message template '%1'?{/ts}
```
[the remark this answers is not in the document]
Comment (eileenmcnaughton) [reply templates/CRM/Admin/Form/MessageTemplates.tpl]: @seamuslee001 agree - @JKingsnorth it's best practice (although sadly not that prevalent) to add |escape to any string in a tpl where it doesn't need to have html in it. For security purposes it's probably important here as the $msg_title could in theory hold script tags

[the remark this answers is not in the document]
Comment (JKingsnorth) [reply templates/CRM/Admin/Form/MessageTemplates.tpl]: @seamuslee001 @eileenmcnaughton - I've pinged you both about this on chat, but can summarise here if you like [=

[the remark this answers is not in the document]
Comment (JKingsnorth) [reply templates/CRM/Admin/Form/MessageTemplates.tpl]: @seamuslee001 / @eileenmcnaughton Added the escape

**Comment by eileenmcnaughton** [comment]
@seamuslee001's issue is fixed - merging. This leave us still discussing best practices around |escape elsewhere

**Pull request merged**